components/comments/commentForm.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+
+const CommentForm = ({ title, identifier, urlTitle }) => {
+    return (
+        <div className="row mt-5 pt-5">
+            <div className="col-lg-12">
+                <h2 id="comments" className="mb-2">
+                    Leave a Comment
+                </h2>
+
+                <form id="webup">
```

Comment:
I'm guessing this id is coming form the original HTML, not sure if is a good idea to have ids in the components if we don't have to.